automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/library/DataLibraryPage.scala

```diff
@@ -124,6 +122,41 @@ class DataLibraryPage(implicit webDriver: WebDriver) extends BaseFireCloudPage
     Label(tagTestId).isVisible
   }
 
+
+  /**
+    * type a string and select from Select
+    *
+    */
+  def doTagsSearch(tag: String): List[Map[String, String]] = {
+    // first clear Tags inputfield
+    clearTag()
+
+    // enter text in Tags inputfield
+    val liOption = "span.select2-container--open ul.select2-results__options li"
```

Comment:
Can this and some of the other queries be pushed into `SearchField`? Also, we should put some `data-test-id`s on the elements if we can... but we might not be able to since I'm guessing it's a 3rd party component. At least if it's pushed into `SearchField` it will all be in one place.

Reply:
I just realized that I might be assuming this is related to `SearchField` but maybe it's not. Anyway, the queries here look complicated enough that they should be simplified or encapsulated in a component of some sort. I should run this test for myself to better understand what these queries are locating.